doc/source/examples/dem/bunny-drill/bunny-drill.rst

```diff
@@ -49,19 +49,19 @@ The mesh is a cylinder generated using the deal.II grid generator.
 Insertion Info
 ~~~~~~~~~~~~~~~~~~~
 
-An insertion box is defined inside at the top part of the cylinder. The insertion region is sufficiently high to ensure that there are no collisions between the particles bouncing on the bottom wall (or particles accumulated at the bottom) and the ones being inserted.
+An insertion plane is defined just bellow the bunny drill. The insertion plane is a useful mechanism to insert particles in a simulation in which the available volume is limited. To ensure a more rapid insertion, we also give an initial velocity to the particles.
 
 .. code-block:: text
 
   subsection insertion info
-    set insertion method                               = non_uniform
-    set inserted number of particles at each time step = 10000
-    set insertion frequency                            = 100000
-    set insertion box points coordinates               = 0.1, -0.07, -0.07 : 0.24, 0.07, 0.07
-    set insertion distance threshold                   = 1.10
-    set insertion maximum offset                       = 0.10
+    set insertion method                               = plane
+    set inserted number of particles at each time step = 200
+    set insertion frequency                            = 4000
+    set insertion plane point                          = 0.025, 0, 0
+    set insertion plane normal vector                  = -1, 0, 0
+    set insertion maximum offset                       = 0.00
```

Comment:
I think this parameter should have a value not equal to zero... 

Otherwise, particles should perfectly bounce on top of each other....

Reply:
It's not a problem because of how chaotic the bouncing is. The packing generated is really not alligned

Reply:
> Do we need this parameter? Is it default?

It's maybe the default value, but we do need to put it to zero, otherwise the insertion is a mess to insert rapidly.